HeterogeneousCore/CUDAUtilities/src/exitSansCUDADevices.cc

```diff
@@ -7,13 +7,15 @@
 
 void exitSansCUDADevices() {
   int devices = 0;
-  auto status = cudaGetDeviceCount(& devices);
+  auto status = cudaGetDeviceCount(&devices);
   if (status != cudaSuccess) {
-    std::cerr << "Failed to initialise the CUDA runtime, the test will be skipped." << "\n";
+    std::cerr << "Failed to initialise the CUDA runtime, the test will be skipped."
+              << "\n";
```

Comment:
why ?

Reply:
No idea. On the other hand the `\n` can be included in the preceding string literal.